src/util/std_expr.h

```diff
@@ -3668,8 +3496,7 @@ inline void validate_expr(const member_designatort &value)
 }
 
 
-/*! \brief Operator to update elements in structs and arrays
-*/
+/// \brief Operator to update elements in structs and arrays
```

Comment:
What is the difference to with_exprt?

Reply:
I don't know. It doesn't seem to be used either. Should it be removed too?